### DISCUSSION TITLE
页面缩放导致table固定列与非固定列出现对不齐

### DISCUSSION THREAD

**Discussion by Xfront-end** [ant-design/ant-design#38457] [Q&A]
![image](https://user-images.githubusercontent.com/52854625/200748536-d0f00c0f-1fd7-4262-88c7-78280af40669.png)

antd版本3.20.x, 官网缩放后也存在同样问题，有无hack方法可以修复一下呢？<html>
<body>

Environment | Info
-- | --
antd | undefined
React | 16.13.1
System | win11
Browser | chrome

</body>
</html>

**Comment by zunyan** [comment]
提供一下复现代码

**Reply by Xfront-end** [reply]
https://codepen.io/Xfront-end/pen/PoaWpKb?editors=0110
